lib/internal/child_process.js

```diff
@@ -516,6 +516,12 @@ function setupChannel(target, channel) {
     // a message.
     target._send({ cmd: 'NODE_HANDLE_ACK' }, null, true);
 
+    // It can happen the handle is not received because of some error on
```

Comment:
Can you replace "It can happen the handle" with "It is possible that the handle"